Source/ObjectiveDropboxOfficial/Shared/Handwritten/OAuth/DBSDKKeychain.m

```diff
@@ -439,13 +435,14 @@ + (void)convertV1TokenToV2:(NSMutableArray<NSArray<NSString *> *> *)v1TokensData
 
   // wait for all token conversion calls to complete and then update the keychain, and call the response block
   dispatch_group_notify(tokenConvertGroup, dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_HIGH, 0), ^{
-    if (!shouldRetry) {
+    if (shouldRetry == NO) {
       for (NSString *uid in tokenConversionResults) {
         [[self class] storeValueWithKey:uid value:[tokenConversionResults objectForKey:uid]];
       }
-      NSUserDefaults *Defaults = [NSUserDefaults standardUserDefaults];
-      [Defaults setObject:@"YES" forKey:[NSString stringWithFormat:kV1TokenMigrationOccurredKeyBase, appKey]];
+      NSUserDefaults *userDefaults = [NSUserDefaults standardUserDefaults];
+		[userDefaults setBool:YES forKey:[NSString stringWithFormat:kV1TokenMigrationOccurredKeyBase, appKey]];
```

Comment:
Yep, totally my bad here. Our Xcode defaults are different than the project's settings. (Wouldn't it be cool if a project could define its own style guide for stuff like this?)